drivers/pds/lib/utils.go

```diff
@@ -1700,7 +1700,7 @@ func CreateTpccWorkloads(dataServiceName string, deploymentID string, scalefacto
 	return false, errors.New("TPCC run failed.")
 }
 
-//CreateDataServiceWorkloads generates workloads for the given dataservices
+// CreateDataServiceWorkloads generates workloads for the given dataservices
```

Comment:
I believe this is added when you run Makefile.

Reply:
Same here